### ISSUE TITLE
[Higher Level Review]: Visual presentation is used to convey the relationship between values and field names

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#106016]
## What happened?

**Describe what went wrong**:

This bug is from a recent 508 Office accessibility audit on the Higher Level Review form. It's potentially across all our forms, not just one, but that requires more testing.

> WCAG 1.3.1 Information and Relationships | Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.
>
> On the Request A Higher-Level Review | Veterans Affairs screen (navigate to Step4 of 4Review application), visual presentation is used to convey the relationship between values and field names but this information is read as unstructured text by assistive technology. Note that after submitting the form, similar information is read as a list. (New: 03/19/2025)

**Add the steps to reproduce the bug**:

1. Go to: `https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/` while logged in with a test user.
2. Get to the review and submit step: `https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/review-and-submit`.
3. Turn on the JAWS screen reader and navigate through the page with all accordions expanded.
4. Note that the Issues for review section announces definition lists as a list. Everywhere else do not announce the lists.

**Add a screen recording or photo**:

https://github.com/user-attachments/assets/de8c1acc-0a7e-4f75-ae4d-8adefa647d81

You'll note in the video that I test the Review and Submit page, then this [WebAIM page with a simple definition list on it](https://webaim.org/techniques/semanticstructure/#lists). I wanted to see how JAWS announced a definition list that wasn't a VADS component. It doesn't announce the definition list as a list there either.

I'm filing this bug, but it needs more investigation.

## How severe is the problem?

- [x] **Critical**: User is block or significantly hindered from submitting the form, data is lost, or claim and evidence test coverage is missing. Fix ASAP.
- [ ] **Moderate**: User is not blocked, but the happy path is interrupted. Fix after critical bugs.
- [ ] **Minor**: Cosmetic bugs and minor language inconsistencies. Defer to a later sprint.

## What is the desired behavior?

1. _When a user does X, the **visual** result is y._
2. _When a user does X, the **backend** result is y._
3. _When a user does X, the **event logging** result is y._

## Definition of Done

- [ ] Acceptance criteria reviewed and approved by product or design
- [ ] Unit test (or other test coverage) added if possible
- [ ] Logging and monitoring in place

**Comment by davidakennedy** [comment]
I did some more testing on this today:

- I created a [CodePen demo](https://codepen.io/davidakennedy/pen/JojaGEb) to have another way to test. Using this demo with JAWS 2019, nothing announces properly, navigating by virtual cursor, JAWS key or to the list item.
- I asked for someone else to test in an Accessibility Slack community. They were able to hear the description list in the CodePen via all three ways. They used JAWS 2024.

**Comment by coforma-jamie** [comment]
@davidakennedy, I tested using JAWS 2024 and your CodePen demo. I found that `<dl>` announces as expected in Firefox, but not Chrome or Edge.

I also came across a couple of tickets:
- [A ticket from 2024 to JAWS](https://github.com/FreedomScientific/standards-support/issues/850) about this issue
- A [Chromium bug ticket](https://issues.chromium.org/issues/376278217) that hasn't been resolved as of March 2025

**Comment by davidakennedy** [comment]
@coforma-jamie shared this recording of her test, using JAWS 2024.2405.48.

https://github.com/user-attachments/assets/231e9ae3-1207-464f-be89-d3789986fa55

**Comment by davidakennedy** [comment]
@jeana-adhoc tested with another test. On JAWS 2025, description lists announce as a list in Chrome and Edge. JAWS 2023 does not recognize this as a description list.

https://github.com/user-attachments/assets/5123f240-1a58-4563-b3ee-ce7663a9473c

Jeana also tested in 2024.2409.2 in Chrome, and it recognized it as a definition list. Her version is .005 more recent than Jamie's.

From the JAWS changelog:

(2024.2406.121) - “Using CTRL+INSERT+L to display a list of all the lists on a web page now works as expected.” This could be related, and a reason why description lists work in newer versions of JAWS 2024, but not ones older than that.

**Comment by davidakennedy** [comment]
To summarize where this issue is at:

- The issue calls out "visual presentation is used to convey the relationship between values and field names but this information is read as unstructured text by assistive technology." We use description lists (dl HTML element) to convey that relationship on the review and submit page.
- The dl elements aren't being announced by older versions of JAWS, by our testing versions older than 2024.2409.2. JAWS does not announce the fact that the element is a description list when using read all, the virtual cursor or list shortcut.
- This doesn't happen in any other major screen readers, including VoiceOver, NVDA and Narrator. Each screen reader announces the items differently. Narrator announce the number of items, but not that it's a list.

We have a few options:

1. We can choose not to support older versions of JAWS, and leave the current code as it exist now.
2. We can change the code on the review and submit page to something else: Likely an ordered or unordered list is the next best option.

Currently, the Veteran Facing Forms team has a redesign of the review and submit page in its backlog.

We may also want to reach out to the 508 Office to get their take on priority if it's happening in older versions of JAWS vs. newer versions.
